clang/test/Driver/sycl-device.cpp

```diff
@@ -7,3 +7,17 @@
 // RUN:   %clang -### -fsycl-device-only %s 2>&1 \
 // RUN:   | FileCheck -check-prefix=CHECK-SYCL-DEV %s
 // CHECK-SYCL-DEV: "-fsycl-is-device"{{.*}} "-internal-isystem" "{{.*}}bin{{[/\\]+}}..{{[/\\]+}}include{{[/\\]+}}sycl"
+
+/// Check that "-Wno-sycl-strict" is set on compiler invocation with "-fsycl"
+/// or "-fsycl-device-only" or both:
```

Comment:
please check also that the same option is added to arguments for NV target: you can dd the check to https://github.com/intel/llvm/blob/sycl/clang/test/Driver/sycl-offload-nvptx.cpp

Reply:
Done